RELEASE_NOTES.md

```diff
@@ -2,6 +2,17 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+## 0.16.3 (October 17, 2023)
+
+- 🧩 Added `release-version` option to `shorebird patch [android|ios-alpha]` to allow specifying a release version.
+- 🩹 Better support for custom Android SDK/java locations.
```

Comment:
I think we have console changes too?  Felix fixed the graphs, and is about to ship collaborators?

Reply:
I mean we have in the past 🤷 .  Doesn't really matter.  But I do think users are going to engage with shorebird at a "release" level not a "repository" level, and so it makes sense for us to write down our "release" level changes somewhere.  That doesn't have to be here, but has been historically.